Agent.Installer.Win/ViewModels/MainWindowViewModel.cs

```diff
@@ -392,7 +392,7 @@ private async Task ExtractEmbeddedServerData()
                 return;
             }
 
-            var embeddedData = await _embeddedDataReader.TryGetEmbeddedData(filePath);
+            var embeddedData = await _embeddedDataReader.TryGetEmbeddedData(filePath!);
```

Comment:
The null check above should set `filePath` to not-null, but wasn't working.  It might be because I'm using WPF in .NET Framework here and setting the language version to 10.

This project will be replaced in the future by .NET 8 a command-line installer or Avalonia app, and it will work for all operating systems.